samples/excel/85-preview-apis/pivottable-source-data.yaml

```diff
@@ -0,0 +1,138 @@
+              ["A Farms", "Lime", "Organic", 300, 2000],
+              ["A Farms", "Lemon", "Organic", 250, 1800],
+              ["A Farms", "Orange", "Organic", 200, 2200],
+              ["B Farms", "Lime", "Conventional", 80, 1000],
+              ["B Farms", "Lemon", "Conventional", 75, 1230],
+              ["B Farms", "Orange", "Conventional", 25, 800],
+              ["B Farms", "Orange", "Organic", 20, 500],
+              ["B Farms", "Lemon", "Organic", 10, 770],
+              ["B Farms", "Kiwi", "Conventional", 30, 300],
+              ["B Farms", "Lime", "Organic", 50, 400],
+              ["C Farms", "Apple", "Organic", 275, 220],
+              ["C Farms", "Kiwi", "Organic", 200, 120],
+              ["D Farms", "Apple", "Conventional", 100, 3000],
+              ["D Farms", "Apple", "Organic", 80, 2800],
+              ["E Farms", "Lime", "Conventional", 160, 2700],
+              ["E Farms", "Orange", "Conventional", 180, 2000],
+              ["E Farms", "Apple", "Conventional", 245, 2200],
+              ["E Farms", "Kiwi", "Conventional", 200, 1500],
+              ["F Farms", "Kiwi", "Organic", 100, 150],
+              ["F Farms", "Lemon", "Conventional", 150, 270]
+            ];
+
+            const range = dataSheet.getRange("A1:E21");
+            range.values = data;
+            range.format.autofitColumns();
+
+            // Create the PivotTable.
+            const pivotTable = totalPivot.pivotTables.add("All Farm Sales", "Data!A1:E21", "TotalPivot!A2");
+            pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Farm"));
+            pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Type"));
+            pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold Wholesale"));
+            pivotTable.dataHierarchies.add(pivotTable.hierarchies.getItem("Crates Sold at Farm"));
+
+            // Switch to the worksheet with a PivotTable.
+            totalPivot.activate();
+
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */ async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+          <p>This sample shows how to get information about the data source of a PivotTable. It returns the type and string representation of the data source.</p>
+          <p><em>Note: This sample works in Excel on Windows and Excel on the web. It doesn't work in Excel on Mac; this is a known issue.</em></p>
+        </section>
+
+        <section class="setup ms-font-m">
+          <h3>Set up</h3>
+          <p> This creates a data sheet and a PivotTable.</p>
```

Comment:
Not sure this label adds much. The term "data sheet" might not convey the relationship between the two worksheets and a more verbose description would be excessive.

Reply:
Good points. I'll remove it.